src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -279,6 +279,16 @@ internal static void CheckForSevereException(Exception e)
                         throw hostException;
                     }
 
+                    // For NanoServer:
+                    // 1. There is no GetCurrentConsoleFontEx / SetCurrentConsoleFontEx on NanoServer;
+                    // 2. We don't handle CJK locales on NanoServer due to lack of win32 API supports on NanoServer.
+#if !CORECLR
```

Comment:
My request is for you to improve the existing comment since you are touching that bit of code